src/Main.java

```diff
@@ -0,0 +1,21 @@
+public class Main {
+    public static void main(String[] args) {
+
+        int i = 1;
+
+        while (i <= 100) {
+            if  (((i % 3) == 0) & ((i % 5)) == 0) {
```

Comment:
bitte immer das und mit kurzschlussauswertung verwenden. und wenn du eine schleife mit zählervariable brauchst, benutz eine for schleife dafür ist sie da :)